src/main/java/mate/academy/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,49 @@
+package mate.academy.dao.impl;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import mate.academy.dao.OrderDao;
+import mate.academy.db.Storage;
+import mate.academy.lib.Dao;
+import mate.academy.model.Order;
+import mate.academy.model.ShoppingCart;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Order create(ShoppingCart shoppingCart) {
+        Order order = new Order(shoppingCart.getUserId());
```

Comment:
I guess it is better to create new Order in Service. In DAO you should only add it in DB.

Reply:
done